src/effects.c

```diff
@@ -445,7 +445,9 @@ void libxmp_process_fx(struct context_data *ctx, struct channel_data *xc, int ch
 			fxp <<= 4;
 			goto fx_setpan;
 		case EX_RETRIG:		/* Retrig note */
+		#ifndef LIBXMP_CORE_PLAYER
 		    fx_retrig:
+		#endif
```

Comment:
Please don't indent these macros; it makes them look like they're part of regular code flow and libxmp fairly consistently keeps them at indentation level 0 in other places.

Reply:
de-indented :)